### DISCUSSION TITLE
with sd_webui_forge

### DISCUSSION THREAD

**Discussion by marc2608** [light-and-ray/sd-webui-replacer#83] [Q&A]
Hi, 
I installed Replacer in Forge, with segment-anything (which works perfectly), I installed the requested models, checkpoint inpainting, etc... But Replacer doesn't work. Is it compatible with Forge? 
Thank you very much in advance for your answers.

**Comment by light-and-ray** [comment]
I've discontinued forge support because it's author stopped it's development for a long time, my PR with compatibility fixes for forge is not still merged despite controlnet extension author has approved it

And also the current webui 1.10 dev branch has the same or even better performance optimization
